Workbooks/UsageMetrics/Usage - MoreEvents/Usage - MoreEvents.workbook

```diff
@@ -512,7 +512,7 @@
       "type": 3,
       "content": {
         "version": "KqlItem/1.0",
-        "query": "let events = dynamic([{userEventsList}]); let mainTable = union {Whoused}\r\n | where timestamp {TimeRange}\r\n | extend name =replace(\"\\n\", \"\",name)\r\n  | where '*' in (events) or name in (events)\r\n {filters};\r\n let queryTable = {mainTableQuery};\r\n let cohortedTable = queryTable\r\n | extend  dimension = column_ifexists('{propertyColumn2:escape}', '')\r\n | extend dimension = iif(isempty(dimension) , \"<undefined>\", dimension)\r\n  | summarize hll = hll(itemId) by tostring(dimension)\r\n | extend Events = dcount_hll(hll)\r\n  | order by Events desc\r\n | serialize rank = row_number()\n | extend dimension = iff(rank > 5, 'Other', dimension)\r\n | summarize merged = hll_merge(hll) by tostring(dimension)\r\n | project ['{column2Name}'] = dimension, Counts = dcount_hll(merged);\r\n cohortedTable",
```

Comment:
why did column_ifexists not work here?  are you escaping the name of the custom property elsewhere so it didn't match up here?

Reply:
Revert to previous version for this query and update dimension escape in the other query